docs/outlook/autolaunch.md

```diff
@@ -0,0 +1,217 @@
+</VersionOverrides>
+```
+
+Outlook on Windows uses a JavaScript file, while Outlook on the web uses an HTML file that references the same JavaScript file. You must provide references to both these files in the manifest as the Outlook platform ultimately determines whether to use HTML or JavaScript based on the Outlook client. As such, to configure event handling, provide the location of the HTML in the `Runtime` element, then in its `Override` child element provide the location of the JavaScript file inlined or referenced by the HTML.
+
+> [!TIP]
+> To learn more about manifests for Outlook add-ins, see [Outlook add-in manifests](manifests.md).
+
+## Implement event handling
+
+You have to implement handling for your selected events. In this case, you'll add handling for composing new items.
+
+1. From the same quick start project, open the file **./src/commands/commands.js** in your code editor.
+
+1. After the `action` function, insert the following JavaScript functions.
+
+    ```js
+    function onMessageComposeHandler(event) {
+      setSubject();
+      event.completed();
+    }
+    function onAppointmentComposeHandler(event) {
+      setSubject();
+      event.completed();
+    }
+    function setSubject() {
+      Office.context.mailbox.item.subject.setAsync("Set by an event-based add-in!");
+    }
+    ```
+
+1. At the end of the file, add the following statements.
+
+    ```js
+    g.onMessageComposeHandler = onMessageComposeHandler;
+    g.onAppointmentComposeHandler = onAppointmentComposeHandler;
+    ```
+
+## Try it out
+
+1. Run the following command in the root directory of your project. When you run this command, the local web server will start (if it's not already running).
+
+    ```command&nbsp;line
+    npm run dev-server
+    ```
+
+1. Follow the instructions in [Sideload Outlook add-ins for testing](sideload-outlook-add-ins-for-testing.md) to sideload the add-in in Outlook.
+
+1. In Outlook on the web, create a new message.
+
+    ![A screenshot of a message window in Outlook on the web with the subject set on compose.](../images/outlook-web-autolaunch.png)
+
+## Event-based activation behavior and limitations
+
+Add-ins that activate based on events are designed to be short-running, up to 330 seconds only. We recommend you have your add-in call the `event.completed` method to signal it has completed processing the launch event. The add-in is also ended when the user closes the compose window.
+
+If the user has multiple add-ins that subscribed to the same event, the Outlook platform launches the add-ins in no particular order. Currently, only 5 event-based add-ins can be actively running. Any additional add-ins are pushed to a queue then run as previously active add-ins are completed or deactivated.
+
+The user can switch or navigate away from the item (that is, moves away from current mail item) where the add-in started running. The add-in that was launched will finish its operation in the background.
```

Comment:
```suggestion
The user can switch or navigate away from the current mail item where the add-in started running. The add-in that was launched will finish its operation in the background.
```
I think the paren says the same thing and adds confusion. Consider deleting or rewording.